astroquery/jplspec/__init__.py

```diff
@@ -1,33 +1,33 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
-JPL Spectral Catalog
---------------------
+JPL Spectral Catalog (Deprecated Location)
+-------------------------------------------
 
+.. deprecated:: 0.4.8
+    The `astroquery.jplspec` module has been moved to `astroquery.linelists.jplspec`.
+    Please update your imports to use `from astroquery.linelists.jplspec import JPLSpec` instead.
+    This backward compatibility layer will be removed in a future version.
 
-:author: Giannina Guzman (gguzman2@villanova.edu)
-:author: Miguel de Val-Borro (miguel.deval@gmail.com)
-
-"""
-from astropy import config as _config
+This module provides backward compatibility for the old import location.
+The JPLSpec module has been reorganized under the linelists subpackage.
 
+For new code, please use::
 
-class Conf(_config.ConfigNamespace):
-    """
-    Configuration parameters for `astroquery.jplspec`.
-    """
-    server = _config.ConfigItem(
-        'https://spec.jpl.nasa.gov/cgi-bin/catform',
-        'JPL Spectral Catalog URL.')
-
-    timeout = _config.ConfigItem(
-        60,
-        'Time limit for connecting to JPL server.')
+    from astroquery.linelists.jplspec import JPLSpec
 
+"""
+import warnings
 
-conf = Conf()
+# Issue deprecation warning
+warnings.warn(
+    "Importing from 'astroquery.jplspec' is deprecated. "
+    "Please use 'from astroquery.linelists.jplspec import JPLSpec' instead. "
+    "The old import path will be removed in a future version.",
+    DeprecationWarning,
+    stacklevel=2
```

Comment:
I don't know if this is necessary or right.  I have no experience playing with stacklevels yet.

Reply:
I can't remember any more why I went with stacklevel 2 for astroquery.irsa, but it worked.